www/js/controller.js

```diff
@@ -814,12 +814,8 @@ define([
 				registry.byId(this.controlMap.value).set('value','');
 				this._model.active.setValue(id, '');
 				domStyle.set('valueInputboxContainer','display','none');
-				this.handleValue(null);
+				//this.handleValue(null);
```

Comment:
I dont know... it was calling PM with a null value as well. I think @dharanisakary was ensuring the container and value in the model was empty and that happens from the lines above. I didnt want to call handler with the null value explicitly and did testing that it doesnt break anything. I dont think anything wrong is happening with removing this piece of code.